examples/time_slider.html

```diff
@@ -26,17 +26,17 @@
         left: 40px;
       }
     </style>
-    <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
+    <link rel="stylesheet" href="https://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
     <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css" />
   </head>
   <body>
     <div id="map"></div>
     <div id="slider"></div>
     <div id="legend"></div>
 
-    
+
     <!-- include cartodb.js library -->
-    <script src="http://libs.cartocdn.com/cartodb.js/v3/3.15/cartodb.js"></script>
+    <script src="https://libs.cartocdn.com/cartodb.js/v3/3.15/cartodb.js"></script>
     <script src="http://code.jquery.com/ui/1.10.3/jquery-ui.js"></script>
```

Comment:
Better use the https + min at: https://code.jquery.com/ui/1.10.3/jquery-ui.min.js